Queueomatic/Client/CustomAuthenticationStateProvider.cs

```diff
@@ -35,7 +35,8 @@ public override async Task<AuthenticationState> GetAuthenticationStateAsync()
             {
                 identity = new ClaimsIdentity(ParseClaimsFromJwt(authToken), "jwt");
                 _httpClient.DefaultRequestHeaders.Authorization =
-                    new AuthenticationHeaderValue("Bearer", authToken.Replace("\"", ""));
+                    new AuthenticationHeaderValue("Bearer", authToken.Replace("\"", "")
+                        .Split(',')[0].Split(':')[1]);
```

Comment:
Why do we split here?

Reply:
Because the old code set the value to `{token:...}` but the new code set the value to _only_ `jwt`. This is done by splitting the deserialized value of `authToken`.